packages/story-editor/src/components/floatingMenu/elements/shapeColor.js

```diff
@@ -17,20 +17,48 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import { useCallback } from '@googleforcreators/react';
+import { trackEvent } from '@googleforcreators/tracking';
 
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import { useStory } from '../../../app';
+import { Color } from './shared';
 
 function ShapeColor() {
+  const { backgroundColor, updateSelectedElements } = useStory(
+    ({ state, actions }) => ({
+      backgroundColor: state.selectedElements[0].backgroundColor,
+      updateSelectedElements: actions.updateSelectedElements,
+    })
+  );
+  const pushUpdate = useCallback(
+    (value) => {
+      trackEvent('floating_menu', {
+        name: 'set_shape_color',
+        element: 'shape',
+      });
```

Comment:
Saw that we have some tracking in floating menus, added this here. Not sure if we want to this or not.

Reply:
Miina made a ticket to fill in the tracking so you adding it here is great, saves us the effort on this action later. https://github.com/GoogleForCreators/web-stories-wp/issues/10871